python/ray/train/torch/torch_detection_predictor.py

```diff
@@ -96,39 +96,21 @@ def _predict_numpy(
             torch.as_tensor(image, dtype=dtype).to(self.device) for image in images
         ]
         outputs = self.call_model(inputs)
-        outputs = _convert_outputs_to_ndarray_batch(outputs)
+        outputs = _convert_outputs_to_batch(outputs)
         outputs = {"pred_" + key: value for key, value in outputs.items()}
 
         return outputs
 
 
-def _convert_outputs_to_ndarray_batch(
+def _convert_outputs_to_batch(
```

Comment:
This is part of a stacked PR #35829

Reply:
you can change the base of this PR to the other PR to remove these changes from review